File: 1904.cpp
#include <iostream>
using namespace std;

void test(int a, int b)
{
    if (a % 2 == 1)
    {
        if (a <= b)
        {
            cout << a << "\t";
            a += 2;
            test(a, b);
        }
    }
    else
    {
        a += 1;
        if (a <= b)
        {
            cout << a << "\t";
            a += 2;
            test(a, b);
        }
    }
}

int main()
{
    int a, b;
    cin >> a >> b;

    test(a, b);

    return 0;
}

File: 1904.py
def test(a, b):
    if (a % 2 == 1):
        if(a <= b):
            print(a, end=" ")
            a += 2
            test(a, b)
    else:
        a += 1
        if(a <= b):
            print(a, end=" ")
            a += 2
            test(a, b)


a, b = input().split()
a = eval(a)
b = eval(b)

test(a, b)
